test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -848,6 +848,8 @@ foo``u{2195}abc
                 #Hexadecimal
                 @{ Script = "0x0u"; ExpectedValue = "0"; ExpectedType = [uint] }
                 @{ Script = "0x41u"; ExpectedValue = "65"; ExpectedType = [uint] }
+                @{ Script = "0xFFFFu"; ExpectedValue = "65535"; ExpectedType = [uint] }
+                @{ Script = "0xFFFFFFFFu"; ExpectedValue = "4294967295"; ExpectedType = [uint] }
```

Comment:
What about 0xFF and 0xFFu?

Reply:
We can add tests for them all if you like, but `0xFF` and `0xFFu` are of equivalent value, just like `0xFFFF` and `0xFFFFu`

Reply:
Please add tests (we need full set of tests) and update the PR description.

Reply:
I think that should about cover the primary differences there. Let me know if I left any gaps, but I think I covered them all this time around.